Integrations/EWSv2/EWSv2.py

```diff
@@ -1555,6 +1560,30 @@ def get_autodiscovery_config():
     }
 
 
+def mark_item_as_read(item_ids, operation='read', target_mailbox=None):
+    marked_items = []
+    account = get_account(target_mailbox or ACCOUNT_EMAIL)
+    if type(item_ids) != list:
+        item_ids = item_ids.split(",")
+
+    items = get_items_from_mailbox(account, item_ids)
+    items = [x for x in items if isinstance(x, Message)]
+
+    for item in items:
+        item.is_read = True if operation == 'read' else False
+        item.save()
+
+        marked_items.append({
+            ITEM_ID: item.item_id,
+            MESSAGE_ID: item.message_id,
+            ACTION: 'marked-as-%s' % operation
+        })
+
+    return get_entry_for_object('Marked items (%s marked operation)' % operation,
```

Comment:
need to add the mail status (read, unread) as part of the context as requested in the comments of the issue.
this output is relevant for all mail related command, for example: `ews-get-items`

Reply:
@yaakovi this is separate issue. We want to push it the same pr or separate one?